tests/serial/convergence-measures/testConvergenceMeasures3.xml

```diff
@@ -44,12 +44,7 @@
     <time-window-size value="1.0" />
     <exchange data="Data1" mesh="MeshOne" from="SolverOne" to="SolverTwo" />
     <exchange data="Data2" mesh="MeshOne" from="SolverTwo" to="SolverOne" />
-    <max-iterations value="100" />
+    <max-iterations value="4" />
```

Comment:
```suggestion
    <max-iterations value="100" />
    <min-iterations value="2" />
```
Why `max-iterations value="4" />`? I wouldn't change the test here, if it is not necessary.

Reply:
The test will SEGFAULT with more than 4 iterations. Given that the test will just run until the end, this should lead to the same result.